static/app/views/explore/metrics/metricPanel/index.tsx

```diff
@@ -25,6 +40,41 @@ export function MetricPanel({traceMetric, queryIndex}: MetricPanelProps) {
     enabled: Boolean(traceMetric.name),
   });
 
+  const columns = TraceSamplesTableColumns;
+  const fields = columns.filter(c => getMetricTableColumnType(c) !== 'stat');
+
+  const metricSamplesTableResult = useMetricSamplesTable({
+    disabled: !traceMetric?.name,
+    limit: RESULT_LIMIT,
+    traceMetric,
+    fields,
+    ingestionDelaySeconds: TWO_MINUTE_DELAY,
+  });
+
+  const metricAggregatesTableResult = useMetricAggregatesTable({
+    enabled: Boolean(traceMetric.name),
+    limit: RESULT_LIMIT,
+    traceMetric,
+  });
+
+  const mode = useQueryParamsMode();
+  const sortBys = useQueryParamsSortBys();
+  const aggregateSortBys = useQueryParamsAggregateSortBys();
+  const [interval] = useChartInterval();
+  const topEvents = useTopEvents();
+
+  useMetricsPanelAnalytics({
```

Comment:
This is going to fire once per panel and again when the query changes for the panel. Is this the intended behaviour?